charts/rancher-webhook/1.0.5+up0.2.6-rc5/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/namespace: cattle-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/release-name: rancher-webhook
```

Comment:
I'm not sure that it makes sense to ask @KevinJoiner to add this as part of this PR - especially when the RC has been cut as is. If I create an issue in the webhook so we can get these added on, can we move this PR forward and get another one going fro the version constraints?

Reply:
@KevinJoiner @MbolotSuse If you need this to move forward you can create the issue and add it later, I'll approve. However, it should ideally get added in the next RC since we are adding these to all charts for this release.